jenkins.sh

```diff
@@ -17,12 +17,12 @@ ls -1 | grep -v ^src | xargs -I{} mv {} src/$PROJ/
 go get github.com/tools/godep
 export PATH=$PATH:$WORKSPACE/bin
 cd src
-godep get -v github.com/onsdigital/rm-survey-service
+godep get -v github.com/ONSdigital/rm-survey-service
 
 # Build the project
-cd github.com/onsdigital/rm-survey-service
+cd github.com/ONSdigital/rm-survey-service
 make build
 
 # Create TAR file and upload to Artifactory
 tar -cvf surveysvc.tar build/linux-amd64/bin/*
-curl -u build:$PASSWORD -X PUT "http://artifactory.rmdev.onsdigital.uk/artifactory/libs-snapshot-local/uk/gov/ons/ctp/product/surveysvc/surveysvc-SNAPSHOT-$BUILD_NUMBER.tar" -T surveysvc.tar
\ No newline at end of file
+curl -u build:$PASSWORD -X PUT "http://artifactory.rmdev.ONSDigital.uk/artifactory/libs-snapshot-local/uk/gov/ons/ctp/product/surveysvc/surveysvc-SNAPSHOT-$BUILD_NUMBER.tar" -T surveysvc.tar
```

Comment:
Correct capitalisation `ONSdigital`

Reply:
thats just the url of the artifactory thats being used, so can't be changed here